CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## [Unreleased]
```

Comment:
There was no way you could have known this but for the non-main branch, I prefer that you don't add a changelog entry. This makes rebasing the v6 branch with the main branch easier. (fewer conflicts).